content/automate/ManageIQ/Transformation/Common.class/__methods__/assesstransformation.rb

```diff
@@ -5,19 +5,18 @@ module Common
         class AssessTransformation
           SUPPORTED_SOURCE_EMS_TYPES = ['vmwarews'].freeze
           SUPPORTED_DESTINATION_EMS_TYPES = ['rhevm'].freeze
-          REQUIRED_CUSTOM_ATTRIBUTES = {
-            'rhevm' => %i(rhv_export_domain_id rhv_cluster_id rhv_storage_domain_id)
-          }.freeze
 
           def initialize(handle = $evm)
             @handle = handle
+            @task ||= ManageIQ::Automate::Transformation::Common::Utils.task(@handle)
```

Comment:
@fdupont-redhat The initialize is called once per instance so you dont need the ||= it should just be
```
@task = ManageIQ::Automate::Transformation::Common::Utils.task(@handle)
```
also for source_vm

Reply:
Good catch.